Easydict/Swift/View/SettingView/Tabs/TabView/AdvancedTab.swift

```diff
@@ -61,6 +61,26 @@ struct AdvancedTab: View {
                     )
                 }
             }
+            Section {
+                Toggle(isOn: $enableHTTPServer) {
+                    AdvancedTabItemView(
+                        color: .green,
+                        systemImage: "network",
+                        labelText: "setting.general.advance.enable_http_server"
+                    )
+                }
+                TextField(text: $httpPort, prompt: Text("8080")) {
+                    AdvancedTabItemView(
+                        color: .red,
+                        systemImage: "externaldrive.connected.to.line.below",
+                        labelText: "setting.general.advance.http_port",
+                        subtitleText: "setting.general.advance.http_port_desc"
+                    )
+                }
+
+            } header: {
```

Comment:
```suggestion

```

I would suggest not having a header for the advanced settings since these are already advanced features for power users to discover

Reply:
This header indicates that this is a special `HTTP Server` Section.

<img width="1320" alt="9233" src="https://github.com/user-attachments/assets/006f19c6-de23-4293-b2a7-df918cbaea73">

Reply:
I understand your idea, but I still suggest keeping this header.

Firstly, I want to use the header to distinguish it from the previous simple combined Section, indicating that this is a separate HTTP Server Section, where all the options inside are related to the HTTP Server, although there are only two now, but there may be more in the future. Adding a header makes it more intuitive and pretty.

> It would save space for more items in the future.

We don't need to worry too much about this, there will be more advanced options later, and it is scrollable so don't worry about not having enough space to display.

Advanced settings are for advanced users, for general users, even if they don't notice them, it won't have a big impact.

<img width="820" alt="24939" src="https://github.com/user-attachments/assets/6a69b8f8-c7be-4ec6-b6e9-ae9a4dd11c46">

<img width="820" alt="72933" src="https://github.com/user-attachments/assets/dfb0c3c5-8cf6-44f3-861d-4c11e0dec121">